packages/runtime/datastore-definitions/src/dataStoreRuntime.ts

```diff
@@ -122,7 +122,7 @@ export interface IFluidDataStoreRuntime
 	 * @param content - Content of the signal.
 	 * @param targetClientId - When specified, the signal is only sent to the provided client id.
 	 */
-	submitSignal(type: string, content: any, targetClientId?: string): void;
+	submitSignal(type: string, content: unknown, targetClientId?: string): void;
```

Comment:
We should use the `submit: (...) => ...` syntax for interfaces as we did in #20251 to help ensure implementations match requirements.